torchrl/envs/libs/robohive.py

```diff
@@ -233,6 +233,7 @@ def register_visual_env(cls, env_name, cams):
 
             if not len(cams):
                 raise RuntimeError("Cannot create a visual envs without cameras.")
+            cams = [i.replace("A:", "A_") for i in cams]
```

Comment:
Let me know if this is something you think should be changed at RoboHive's end.

Reply:
Ah, I see your comment above. I believe `A:trackingZ` is a valid string for MuJoCo as well as RoboHive.

Reply:
I would encourage using environment (and derived camera etc) names that are conventional and can be used for serialization. I can imagine someone using the camera name within the experiment name and use the experiment name to save things on disk

Reply:
I have name a note of this for the next release.